src/commands/phpmyadmin.ts

```diff
@@ -159,14 +163,30 @@ export class PhpMyAdminCommand {
 		return await getPhpMyAdminStatus( this.app.id as number, this.env.id as number );
 	}
 
+	async readyToServe(): Promise< boolean > {
+		const url = `https://${ this.env.primaryDomain?.name }/.wpvip/pma/auth`;
+
+		const agent = createProxyAgent( url );
+		const resp = await fetch( url, {
+			method: 'GET',
+			redirect: 'manual',
+			agent: agent ?? undefined,
+		} );
+
+		// If we get 401, the domain is hidden behind a login wall, so we cannot access it
+		if ( resp.status === 401 ) {
+			throw new Error( 'We cannot check if PhpMyAdmin is ready due to a login wall' );
```

Comment:
Won't this bail out of the polling completely?

Reply:
Yes. We don't have a choice on this if the domain itself is hidden behind a login wall.